diesel_cli/src/print_schema.rs

```diff
@@ -51,6 +51,13 @@ pub fn run_print_schema<W: IoWrite>(
     Ok(())
 }
 
+fn simplify_patch_error(err: IoError) -> Box<dyn Error> {
+    match err.kind() {
+        ErrorKind::NotFound => "Unable to find 'patch' command, is it installed?".into(),
```

Comment:
IMO: I'm not sure but using backticks instead of single quotations would be good?

Reply:
I don't really have a strong preference for either. The only intention here is to make sure the user doesn't read it as the word *patch* instead of the command `patch`. I've updated the commit to use backticks.